README.md

```diff
@@ -5,10 +5,90 @@
 [![tests](https://github.com/FFY00/mesonpy/actions/workflows/tests.yml/badge.svg)](https://github.com/FFY00/mesonpy/actions/workflows/tests.yml)
 [![codecov](https://codecov.io/gh/FFY00/mesonpy/branch/main/graph/badge.svg?token=xcb2u2YvVk)](https://codecov.io/gh/FFY00/mesonpy)
 
-Meson PEP 517 Python build backend.
+Python build backend ([PEP 517](https://www.python.org/dev/peps/pep-0517/)) for Meson projects.
```

Comment:
How about `PEP 517` -> via `pyproject.toml`? There's multiple PEPs involved, and Pradyun has been on a campaign for people to stop using PEP numbers.

Reply:
My idea was to add the other build system specific PEPs we support here (eg. [PEP 660](https://www.python.org/dev/peps/pep-0660/)), as people will look out for that, but also describe functionalities as we add them for the ones who aren't aware. Does that sound reasonable to you?

Reply:
That sounds very reasonable to me.